online-inference/fastertransformer/download-weights-job.yml

```diff
@@ -0,0 +1,310 @@
+              dims: [ -1 ]
+            },
+            {
+              name: "output_log_probs"
+              data_type: TYPE_FP32
+              dims: [ -1, -1 ]
+            }
+          ]
+          instance_group [
+            {
+              count: 1
+              kind: KIND_CPU
+            }
+          ]
+          parameters {
+            key: "tensor_para_size"
+            value: {
+              string_value: "1"
+            }
+          }
+          parameters {
+            key: "pipeline_para_size"
+            value: {
+              string_value: "1"
+            }
+          }
+          parameters {
+            key: "max_seq_len"
+            value: {
+              string_value: "528"
+            }
+          }
+          parameters {
+            key: "is_half"
+            value: {
+              string_value: "1"
+            }
+          }
+          parameters {
+            key: "head_num"
+            value: {
+              string_value: "16"
+            }
+          }
+          parameters {
+            key: "size_per_head"
+            value: {
+              string_value: "256"
+            }
+          }
+          parameters {
+            key: "inter_size"
+            value: {
+              string_value: "16384"
+            }
+          }
+          parameters {
+            key: "vocab_size"
+            value: {
+              string_value: "50400"
+            }
+          }
+          parameters {
+            key: "start_id"
+            value: {
+              string_value: "50256"
+            }
+          }
+          parameters {
+            key: "end_id"
+            value: {
+              string_value: "50256"
+            }
+          }
+          parameters {
+            key: "decoder_layers"
+            value: {
+              string_value: "28"
+            }
+          }
+          parameters {
+            key: "model_name"
+            value: {
+              string_value: "gptj-6-b"
+            }
+          }
+          parameters {
+            key: "rotary_embedding"
+            value: {
+              string_value: "64"
+            }
+          }
+          parameters {
+            key: "model_type"
+            value: {
+              string_value: "GPT-J"
+            }
+          }
+          parameters {
+            key: "model_checkpoint_path"
+            value: {
+              string_value: "/mnt/pvc/triton-model-store/fastertransformer/1"
+            }
+          }
+          parameters {
+            key: "enable_custom_all_reduce"
+            value: {
+              string_value: "0"
+            }
+          }' >> /mnt/pvc/triton-model-store/fastertransformer/config.pbtxt
+           
+          rm -rf /mnt/pvc/triton-model-store/fastertransformer/1/1-gpu;
+        volumeMounts:
+          - name: cache
+            mountPath: /mnt/pvc
+        resources:
+          requests:
+            cpu: 16
+            memory: 64Gi 
+            nvidia.com/gpu: 1
```

Comment:
Does the conversion process use GPUs? If it does, a GPU affinity needs to exist, but I don't think it does and there should not be a GPU request.